PokeAlarm/Filters.py

```diff
@@ -140,7 +140,8 @@ def load_egg_section(settings):
     egg = {
         "enabled": bool(parse_boolean(settings.pop('enabled', None)) or False),
         "min_level": int(settings.pop('min_level', 0) or 0),
-        "max_level": int(settings.pop('max_level', 10) or 10)
+        "max_level": int(settings.pop('max_level', 10) or 10),
+        "contains": settings.pop('gymname_contains', [])
```

Comment:
We should probably do some check to ensure this is a list. If it's a string bad things will happen.

Reply:
Can you recommend if/how I could do this with 'create_multi_filter' as I cannot figure that out?  Or should I just just create a separate function for that?

Reply:
I don't think you should use create_mutli_filter. The safest bet is to just check if it's a list and raise a ValueException afterwards.